src/server/raft_client.rs

```diff
@@ -46,6 +46,7 @@ struct Queue {
     buf: ArrayQueue<RaftMessage>,
     /// A flag indicates whether the queue can still accept messages.
     connected: AtomicBool,
+    paused: AtomicBool,
```

Comment:
It can be merged with `connected`.

Reply:
Yes, what I mean is using an `AtomicU8` to merge `connected` and `paused`. Apparently, a connection can either be connected, paused or disconnected (should never connect again).

Reply:
ok